float32 min_angle

---
bool result
